- Documentation for 'rsize' and 'wsize' NFS mount options is misleading

--- util-linux-2.12p/mount/mount.8.nfsdoc	2005-05-10 11:46:25.119592000 -0400
+++ util-linux-2.12p/mount/mount.8	2005-10-07 09:46:13.374277000 -0400
@@ -1339,12 +1339,17 @@ For details, see
 
 Especially useful options include
 .TP
-.B rsize=8192,wsize=8192
-This will make your nfs connection faster than with the default
-buffer size of 4096. (NFSv2 does not work with larger values of
-.B rsize
-and
-.BR wsize .)
+.B rsize=32768,wsize=32768
+This causes the NFS client to try to negotiate a buffer size
+up to the size specified.
+A large buffer size does improve performance, but both the 
+server and client have to support it.
+In the case where one of these does not support the size specified, 
+the size negotiated will be the largest that both support.
+.TP
+.B intr
+This will allow NFS operations (on hard mounts) to be 
+interrupted while waiting for a response from the server. 
 .TP
 .B hard
 The program accessing a file on a NFS mounted file system will hang
